packages/page-accounts/src/Contacts/Address.tsx

```diff
@@ -129,48 +105,32 @@ function Address ({ address, className, filter, isFavorite, toggleFavorite }: Pr
     [address, toggleFavorite]
   );
 
-  const _saveName = useCallback(
-    (): void => {
-      toggleEditName();
-
-      const meta = { name: accName, whenEdited: Date.now() };
-
-      if (address) {
-        try {
-          const currentKeyring = keyring.getPair(address);
-
-          currentKeyring && keyring.saveAccountMeta(currentKeyring, meta);
-        } catch (error) {
-          keyring.saveAddress(address, meta);
-        }
-      }
-    },
-    [accName, address, toggleEditName]
-  );
-
-  const _saveTags = useCallback(
-    (): void => {
-      toggleEditTags();
-
-      const meta = { tags, whenEdited: Date.now() };
-
-      if (address) {
-        try {
-          const currentKeyring = keyring.getPair(address);
-
-          currentKeyring && keyring.saveAccountMeta(currentKeyring, meta);
-        } catch (error) {
-          keyring.saveAddress(address, meta);
-        }
-      }
-    },
-    [address, tags, toggleEditTags]
-  );
-
   if (!isVisible) {
     return null;
   }
 
+  const _toggleForget = (): void => setIsForgetOpen(!isForgetOpen);
```

Comment:
These need to be `useCallback`. Same with the functions below. We don't want to re-create them each render. (The removed instances were `useCallback`). The issue is if not, then all the child components will be re-done each and everytime there is a trigger.